lib/include/jxl/encode.h

```diff
@@ -335,6 +335,19 @@ typedef enum {
    */
   JXL_ENC_FRAME_SETTING_JPEG_COMPRESS_BOXES = 33,
 
+  /** Control what kind of buffering is used, when using chunked image frames.
+   * 0 = buffers everything, basically the same as non-streamed code path
+   (mainly for testing)
+   * 1 = can buffer internal data (the tokens)
+   * 2 = can buffer the output
+   * 3 = minimize buffer usage: streamed input and streamed output, writing TOC
+   last (will not work with progressive)
```

Comment:
How will writing the TOC last work? The output writing will be random access?
It's not really "streamed output" if there's going to be seeking involved...

Reply:
Yes.
Should we call it "buffered" instead?

Reply:
Or chunky/chunked?